packages/template-retail-react-app/app/pages/checkout/index.jsx

```diff
@@ -84,7 +84,11 @@ const Checkout = () => {
                     </GridItem>
 
                     <GridItem py={6} px={[4, 4, 4, 0]}>
-                        <OrderSummary showTaxEstimationForm={false} showCartItems={true} />
+                        <OrderSummary
```

Comment:
Need to pass basket as prop into OrderSummary now, otherwise checkout will throw error

Reply:
what error is this about? 🤔